src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Service/SecondFactorMailService.php

```diff
@@ -244,4 +268,28 @@ public function sendVettedEmail(
 
         $this->mailer->send($message);
     }
+
+    /**
+     * @param string $name
+     * @param string $locale
+     * @param string $fallbackLocale
+     * @return null|\Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Dto\EmailTemplate
+     */
+    private function findEmailTemplate($name, $locale, $fallbackLocale)
+    {
+        $emailTemplate = $this->emailTemplateService->findByName($name, $locale, $this->fallbackLocale);
+
+        if ($emailTemplate) {
+            return $emailTemplate;
+        }
+
+        if ($this->warnOnMissingTemplateConfiguration) {
+            $this->logger->warning(
+                'Skipping sending mail because template configuration is missing',
```

Comment:
Oh, if I'm still on time, I think it would be very helpful when scanning the log messages that the email template's name is included in the log message. Sorry!

If you agree, change it, otherwise, merge away :ship: